sdk/synapse/Azure.Analytics.Synapse.AccessControl/samples/Sample1_HelloWorld.md

```diff
@@ -6,64 +6,20 @@ This sample demonstrates basic operations with two core classes in this library:
 
 To interact with Azure Synapse, you need to instantiate a `AccessControlClient`. It requires an endpoint URL and a `TokenCredential`.
 
-```C# Snippet:CreateAccessControlClient
```

Comment:
Snippets will be restored in: https://github.com/azure/azure-sdk-for-net/issues/24395